OnlineShop.Services.Order.BusinessLayer/Infrastructure/Validators/OrderCreateDtoValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using FluentValidation;
+using OnlineShop.Services.Order.BusinessLayer.Models.Dto;
+
+namespace OnlineShop.Services.Order.BusinessLayer.Infrastructure.Validators
+{
+    public class OrderCreateDtoValidator : BaseValidator<OrderCreateDto>
+    {
+        public OrderCreateDtoValidator()
+        {
+            RuleFor(dto => dto.UserId).NotEmpty();
+            RuleForEach(dto => dto.ProductIds).NotEmpty();
+            RuleFor(dto => dto.Total).GreaterThanOrEqualTo(0);
+            RuleFor(dto => dto.PhoneNumber).NotEmpty();
```

Comment:
you can create more specific validation for phone number. set its length and set with regular expression that it could consist only of digits for example